monarchmoney/monarchmoney.py

```diff
@@ -390,6 +390,235 @@ async def get_account_holdings(self, account_id: int) -> Dict[str, Any]:
             variables=variables,
         )
 
+    async def get_budgets(
+        self,
+        start_date: Optional[str] = None,
+        end_date: Optional[str] = None,
+        use_legacy_goals: Optional[bool] = False,
+        use_v2_goals: Optional[bool] = True,
+    ) -> Dict[str, Any]:
+        """
+        Get your budgets and correspond actual amounts from the account.
+
+        :param start_date: the earliest date to get budget data from, in "yyyy-mm-dd" format (default: last month)
+        :param end_date: the latest date to get budget data from, in "yyyy-mm-dd" format (default: next month)
+        :param use_legacy_goals: Set True to return monthly budget set aside for goals (think no longer used)
+        :param use_v2_goals: Set True to return monthly budget set aside for version 2 goals
```

Comment:
minor: can we add a snippet here that explains that, if the `start_date` and `end_date` params aren't set, it will default to retrieving three months of budgets, starting with last month (according to today's date)? Assuming I read the code correctly. :)

Reply:
@grablair Updated docstring to clarify what the default values will be.